bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/BoschSHCHandler.java

```diff
@@ -114,7 +114,7 @@ protected BoschSHCHandler(Thing thing) {
 
     /**
      * Initializes this handler. Use this method to register all services of the device with
-     * {@link #registerService(TService, Consumer<TState>, Collection<String>, boolean)}.
+     * {@link #registerService}.
```

Comment:
There are two issues with this link:
* There are 3 overloaded variants of the method `registerService()`, so the link needs to be qualified. The correct link would look like this: `{@link #registerService(BoschSHCService, Consumer, Collection, boolean)}` (without the generic type arguments)
* I noticed that the clients are not supposed to register their services in `initialize()`, but in the more specific method `initializeServices()`. So the Javadoc is actually misplaced here. So the best solution would be to replace the Javadoc of `initializeServices()` with the sentence `Use this method...` from this Javadoc. I would understand if this is out of scope for this pull request, so you can just fix the link and we can move the Javadoc later if you prefer.

Reply:
Thanks! I tried do do as requested, could you pls check?

Reply:
Perfect, thanks 😎